merkato/utils/database_utils.py

```diff
@@ -151,7 +157,11 @@ def merkato_exists(UUID):
     finally:
         c = conn.cursor()
         c.execute('''SELECT * FROM merkatos WHERE exchange_pair = ?''', (UUID,))
-        result = len(c.fetchall())
+        query = c.fetchall()
```

Comment:
what is currently wrong with this? The print statements are not necessary, and the function works as is to my knowledge.

Reply:
I'll wait on a response before merging

Reply:
that's not a bugfix, I was trying do debug why a GUI merkato thinks it doesn't need to distribute ladders. If you run the simple app and try to make a test exchange merkato, those print statements would show you it thinking the merkato exists already.